metadata.json

```diff
@@ -1,23 +1,79 @@
 {
-  "name": "mrpuppet-tagmail",
-  "version": "0.1.3",
-  "author": "mrpuppet",
-  "summary": "A report processor plugin for re-enabling the tagmail feature in PE 3.7 onwards. Use this to have Puppet send emails based on tags",
+  "name": "puppetlabs-tagmail",
+  "version": "1.0.0",
+  "author": "puppetlabs",
+  "summary": "This module provides a report processor that sends events to email addresses based on tagging of resources",
   "license": "Apache-2.0",
+  "source": "git://github.com/puppetlabs/puppetlabs-tagmail.git",
+  "project_page": "http://links.puppetlabs.com/tagmail-module",
```

Comment:
Yeah, I was following the lead from other internal modules, figured we'd resolve later.